Assets/AnkrDemo/Scripts/DemoContractHandler.cs

```diff
@@ -6,23 +7,27 @@
 using AnkrSDK.Provider;
 using AnkrSDK.Utils;
 using AnkrSDK.WearableNFTExample;
+using Common.Logging.Configuration;
 using Cysharp.Threading.Tasks;
 using Demo.Scripts.Helpers;
 using TMPro;
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace Demo.Scripts
 {
 	public class DemoContractHandler : MonoBehaviour
 	{
 		private const string TransactionGasLimit = "1000000";
 
-		[SerializeField] private TMP_Text _text;
+		[FormerlySerializedAs("_text")] [SerializeField] private TMP_Text text;
```

Comment:
why?

Reply:
uh might be my IDE that did that whenever i tried to rename another variable